TitleEdit/Windows/Tabs/PresetTab.cs

```diff
@@ -827,6 +835,11 @@ private void LoadCurrentTerritory()
             {
                 preset.LocationModel.WeatherId = weathers.FirstOrDefault((byte)2);
             }
+            
+            if (preset.LocationModel is { SaveLayout: true, SaveHousing: true })
+            {
+                Services.HousingService.SetHousing(ref preset.LocationModel);
+            }
```

Comment:
There should be no logic check here and instead the housing values should be reset on save if they are not needed (in the save button specifically).

I do it like this because right now if the user starts creating a new preset they'll need to check the housing checkbox and then press Apply current for it to load, instead of it just working in the background.